### ISSUE TITLE
Support trailing wildcards in manually added disabled site domains

### ISSUE THREAD

**Issue by kaovilai** [EFForg/privacybadger#2644]
User story: Web dev contoso.com is using multiple TLDs in the user flow. Login throws us to contoso.us and shopping cart goes to contoso.shop
Now consider a multi-national company website whose dev ability to redirect users to many ccTLDs is beyond your imagination, you cannot simply block/unblock the naked domain because each ccTLD is a different naked domain. regex is needed because thankfully it all have the same name in front of the TLD extension.

I have encountered broken (or at least I felt like) site with klm.com/en_us as I navigate the site I am thrown around klm.us and some sites are /kr_en different locales etc.
It would be nice to have a regex way maybe in the "uBlock origin's advanced user mode". This is a revival of #741

**Comment by ghostwords** [comment]
It sounds like we should teach Privacy Badger that these domains [belong to the same entity](https://github.com/EFForg/privacybadger/blob/master/doc/fixing-broken-sites.md#domains-that-are-part-of-the-site-but-dont-look-like-it) in order to fix this breakage for all Privacy Badger users.

**Comment by kaovilai** [comment]
> 
> 
> It sounds like we should teach Privacy Badger that these domains [belong to the same entity](https://github.com/EFForg/privacybadger/blob/master/doc/fixing-broken-sites.md#domains-that-are-part-of-the-site-but-dont-look-like-it) in order to fix the breakage for all Privacy Badger users.

Sounds like a feasible solution. But one that each user must report each site. A lot of PRs will be needed long in the future. In contrast, if regex is enabled, a user who encounter that one website won't have to report it if it's not annoying enough.

**Comment by ghostwords** [comment]
Most users never visit the options page. If it's not a simple button, most users will never see it.

**Comment by ghostwords** [comment]
We do already support leading wildcards ("*.example.com"). We could also support trailing wildcards, and there is enough interest in this feature.

**Comment by ghostwords** [comment]
Related to #1843.

**Comment by ghostwords** [comment]
KLM breakages should be fixed by 16fc3f9d2aea958108b163c1f58e8476ffe11197.
